Bottom: 36b85c924788865dc1a0d2a56ab345f727ea243e
Top:    d7dc2e362b1b50c21e88fadd580de4fcfc627357
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-11 12:41:08 -0700

Refresh of fix

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 4683ee53050..6e92b94edef 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -765,6 +765,18 @@ class FileHandleTestCase(lldbtest.TestBase):
         self.assertEqual(f.getvalue().strip().split(), ["foobar", "7"])
 
 
+    @add_test_categories(['pyapi'])
+    def test_stdout_file(self):
+        with open(self.out_filename, 'w') as f:
+            status = self.debugger.SetOutputFile(f)
+            self.assertTrue(status.Success())
+            self.handleCmd(r"script sys.stdout.write('foobar\n')")
+        with open(self.out_filename, 'r') as f:
+            # python2 returns None from write, python3 returns 7
+            lines = [x for x in f.read().strip().split() if x != "7"]
+            self.assertEqual(lines, ["foobar"])
+
+
     @add_test_categories(['pyapi'])
     @skipIf(py_version=['<', (3,)])
     def test_identity(self):
